core/src/main/java/jenkins/model/ModelObjectWithContextMenu.java

```diff
@@ -317,6 +330,14 @@ class MenuItem {
         public boolean requiresConfirmation;
 
 
+        /**
+         * True to require confirmation after a click.
+         * @since TODO
+         */
+        @Exported
+        @SuppressFBWarnings(value = "URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD", justification = "read by Stapler")
+        public int badge;
```

Comment:
this should still be private just add a getter for stapler, historical methods have had the suppression added